src/main/java/org/broadinstitute/hellbender/utils/GenotypeUtils.java

```diff
@@ -136,8 +136,14 @@ public static GenotypeCounts computeDiploidGenotypeCounts(final VariantContext v
         return new GenotypeCounts(genotypeWithTwoRefsCount, genotypesWithOneRefCount, genotypesWithNoRefsCount);
     }
 
+    /**
+     * Do we have (or can we infer) likelihoods necessary for allele frequency calculation?
+     * Some reblocked and/or DRAGEN GVCFs omit likelihoods for ref blocks, but we can estimate them
+     * @param g a genotype of unknown call and ploidy
+     * @return  true if we have enough info for AF calculation
+     */
     public static boolean genotypeIsUsableForAFCalculation(Genotype g) {
-        return g.hasLikelihoods() || g.hasGQ() || g.getAlleles().stream().anyMatch(a -> a.isCalled() && a.isNonReference() && !a.isSymbolic());
+        return g.hasLikelihoods() || (g.isHomRef() && g.hasGQ() && 2 == g.getPloidy());
```

Comment:
This condition has changed quite a bit in this PR! Can you briefly explain?

Reply:
The only scenario where we can (somewhat confidently) reconstruct PLs from GQ is hom-ref, but it has to be diploid (probably we could do haploid and higher ploidies too, but the ROI is low)
The second case isn't useful because if we get there because there are no likelihoods and no GQ, then there's no useful data.